perf-common/src/main/java/com/github/projectflink/common/AnalyzeTool.java

```diff
@@ -53,15 +56,15 @@ public static Result analyze(String file) throws IOException {
 
 		String l;
 		Pattern latencyPattern = Pattern.compile(".*Latency ([0-9]+) ms.*");
-		Pattern throughputPattern = Pattern.compile(".*That's ([0-9.]+) elements\\/second\\/core.*");
+		Pattern throughputPattern = Pattern.compile(".*we received ([0-9]+) elements. That's ([0-9.]+) elements\\/second\\/core.*");
 		Pattern hostPattern = Pattern.compile("Container: .* on ([^.]+).*");
 		Pattern stormHostPattern = Pattern.compile(".*Client environment:host.name=([^.]+).*");
 
 		DescriptiveStatistics latencies = new DescriptiveStatistics();
-		SummaryStatistics throughputs = new SummaryStatistics();
+		WeightedMean throughputs = new WeightedMean();
 		String currentHost = null;
 		Map<String, DescriptiveStatistics> perHostLat = new HashMap<>();
-		Map<String, SummaryStatistics> perHostThr = new HashMap<>();
+		Map<String, WeightedMean> perHostThr = new HashMap<>();
```

Comment:
now this is inconsistent: `perHostThr` should be `perHostThroughputs` (after the change above)

(similar for the latencies)